scd4j/src/main/java/com/datamaio/scd4j/conf/Configuration.java

```diff
@@ -156,21 +160,14 @@ private synchronized String buildCurrentExecutionPath() {
 			SimpleDateFormat df = new SimpleDateFormat("yyyyMMddHHmmss");
 			
 			Path workspace = workspace();
-			Path config = getConfig();
-			String fileName;
-			if(config!=null) {
-				fileName = workspace.relativize(config).toString().replace("config/", "");
-			} else {
-				fileName = workspace.toString().replace("config/", "");
-			}
-			
+			String fileName = this.getConfig().getFileName().toString();
 			if(fileName.endsWith(".conf")) {
 				fileName = fileName.replace(".conf", "");
 			}
 			if(fileName.endsWith(".properties")) {
 				fileName = fileName.replace(".properties", "");
 			}
-			String module = getModule().getFileName().toString();
+			final String module = workspace.relativize(this.getModule().toAbsolutePath()).toString().replace(Configuration.MODULES_FOLDER + File.separator, "");
```

Comment:
I never thought of using multi-level modules. Does it work? 
Have you tried these new changes on the linux server or only on the windows? Have you tried it inside and outside of eclipse?